tardis/model/tests/test_csvy_model.py

```diff
@@ -75,6 +76,26 @@ def test_compare_models(model_config_fnames):
     )
 
 
+def test_dimensionality_after_update_v_inner_boundary(
+    model_config_fnames,
+):
+    """Test that the dimensionality of SimulationState parameters after updating v_inner_boundary
+    in the context of csvy models specifically"""
+    csvy_config_file, _ = model_config_fnames
+    config = Configuration.from_yaml(csvy_config_file)
+    atom_data = AtomData.from_hdf(config.atom_data)
+    csvy_model = SimulationState.from_csvy(config, atom_data=atom_data)
+
+    config.model.v_inner_boundary = csvy_model.velocity[1]
+    new_csvy_model = SimulationState.from_csvy(config, atom_data=atom_data)
+
+    assert new_csvy_model.no_of_raw_shells == csvy_model.no_of_raw_shells
+    assert new_csvy_model.no_of_shells == csvy_model.no_of_shells - 1
+    assert new_csvy_model.velocity.shape[0] == csvy_model.velocity.shape[0] - 1
+    assert new_csvy_model.density.shape[0] == csvy_model.density.shape[0] - 1
+    assert new_csvy_model.volume.shape[0] == csvy_model.volume.shape[0] - 1
+
+
 @pytest.fixture(scope="module")
 def csvy_model_test_abundances(example_csvy_file_dir):
```

Comment:
this fixture can be maybe good to use in the test above?

Reply:
created new fixture in this test.py